test/bin/scenario.sh

```diff
@@ -933,8 +938,18 @@ stress_testing() {
 
 # Run the tests for the current scenario
 run_tests() {
-    local -r vmname="${1}"
-    local -r full_vmname="$(full_vm_name "${vmname}")"
+    local vmname="${1}"
+    local full_vmname
```

Comment:
Related to my comment on not setting `host1` default for the override, we should initialize `full_vmname` like it was before and override when necessary.

Reply:
Done.